src/sagemaker/jumpstart/curated_hub/parsers.py

```diff
@@ -0,0 +1,422 @@
+        else None
+    )
+    document["HostingPrepackedArtifactVersion"] = model_specs.hosting_prepacked_artifact_version
+    document["HostingUseScriptUri"] = model_specs.hosting_use_script_uri
+    document["HostingEulaUri"] = (
+        s3_path_join("s3://", content_bucket, model_specs.hosting_eula_key)
+        if model_specs.hosting_eula_key is not None
+        else None
+    )
+    if model_specs.hosting_model_package_arns and region in model_specs.hosting_model_package_arns:
+        document["HostingModelPackageArn"] = model_specs.hosting_model_package_arns[region]
+    document["DefaultInferenceInstanceType"] = model_specs.default_inference_instance_type
+    document["SupportedInferenceInstanceTypes"] = model_specs.supported_inference_instance_types
+    document["SageMakerSdkPredictorSpecifications"] = model_specs.predictor_specs
+    document["InferenceVolumeSize"] = model_specs.inference_volume_size
+    document["InferenceEnableNetworkIsolation"] = model_specs.inference_enable_network_isolation
+    document["ResourceNameBase"] = model_specs.resource_name_base
+    document["DefaultPayloads"] = {
+        alias: walk_and_apply_json(payload.to_json(), snake_to_upper_camel)
+        for alias, payload in model_specs.default_payloads.items()
+    }
+    document["HostingResourceRequirements"] = model_specs.hosting_resource_requirements
+    document[
+        "HostingInstanceTypeVariants"
+    ] = model_specs.hosting_instance_type_variants.regionalize(region)
+    default_training_dataset_key = studio_specs.get("defaultDataKey")
+    document["DefaultTrainingDatasetUri"] = (
+        s3_path_join("s3://", content_bucket, model_specs.hosting_prepacked_artifact_key)
+        if default_training_dataset_key
+        else None
+    )
+    document["FineTuningSupported"] = studio_specs.get("fineTuningSupported")
+    document["ValidationSupported"] = studio_specs.get("validationSupported")
+    document["MinStudioSdkVersion"] = studio_specs.get("minServerVersion")
+    if studio_specs.get("notebookLocations"):
+        notebook_locations = studio_specs.get("notebookLocations")
+        notebook_location_uris = {}
+        if notebook_locations.get("demoNotebook"):
+            notebook_location_uris["demo_notebook"] = s3_path_join(
+                "s3://", content_bucket, notebook_locations.get("demoNotebook")
+            )
+        if notebook_locations.get("modelFit"):
+            notebook_location_uris["model_fit"] = s3_path_join(
+                "s3://", content_bucket, notebook_locations.get("modelFit")
+            )
+        if notebook_locations.get("modelDeploy"):
+            notebook_location_uris["model_deploy"] = s3_path_join(
+                "s3://", content_bucket, notebook_locations.get("modelDeploy")
+            )
+        document["NotebookLocationUris"] = notebook_location_uris
+
+    document["ModelProviderIconUri"] = None  # Not needed for private beta.
+
+    # TODO: Fetch from studio manifest
```

Comment:
Still needed?

Reply:
Yes since we need it from studio manifest, not studio spec